icons/attach-icon.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import { motion, useAnimation, Variants } from "framer-motion";
+import { useEffect } from "react";
+
+const pathVariants: Variants = {
+  normal: { pathLength: 1, opacity: 1, pathOffset: 0 },
+  animate: {
+    opacity: [0, 1],
+    pathLength: [0, 1],
+    transition: {
+      delay: 0.3,
+      duration: 0.4,
+      opacity: { duration: 0.1, delay: 0.3 },
```

Comment:
also, i think you should remove `duration` or set it to `0.1` because it just slows down the animation, in this case i guess it's unnecessary. it's better to use it when we have an order and we need to stick to it